lib/workers/repository/config-migration/branch/commit-message.ts

```diff
@@ -0,0 +1,40 @@
+import type { RenovateConfig } from '../../../../config/types';
+import { CommitMessage } from '../../model/commit-message';
+
+export class ConfigMigrationCommitMessageFactory {
```

Comment:
looks like we need to wait for the cost message refactoring PR here

Reply:
changed it to not use the commit-message for now, in order not to be blocked.

Reply:
`import { CommitMessage } from '../../model/commit-message';` is gone, donno why it not getting `outdated` and updated.
when watching raw it is gone....

anyway, i guess it will take some time till this gets approved, till then the `model/commit-message` PR will be probably merged and will be used here again.